README.md

```diff
@@ -179,61 +269,31 @@ The `opts` argument can have the following properties:
 
-### `sbot.metafeeds.findOrCreate(metafeed, visit, details, cb)`
 
-_Looks for the first subfeed of `metafeed` that satisfies the condition in
-`visit`, or creates it matching the properties in `details`._
-
-This is strictly concerned with meta feeds and sub feeds that **you own**, not
-with those that belong to other peers.
-
-`metafeed` can be either `null` or a meta feed object `{ seed, keys }` (as
-returned by `getRoot()`). If it's null, then the root meta feed will be created,
-if and only if it does not already exist. If it's null and the root meta feed
-exists, the root meta feed will be returned via the `cb`. Alternatively, you can
-call this API with just the callback: `sbot.metafeeds.findOrCreate(cb)`.
-
-`visit` can be either `null` or a function of the shape
-`({feedpurpose,subfeed,metadata,keys}) => boolean`. If it's null, then one
-arbitrary subfeed under `metafeed` is returned.
-
-`details` can be `null` only if if `metafeed` is null, but usually it's an
-object with the properties:
-
-- `feedpurpose`: any string to characterize the purpose of this new subfeed
-- `feedformat`: the string `'classic'` or the string `'bendybutt-v1'`
-- `metadata`: an optional object containing other fields. One can
-  include `recps` here to encrypt the message
-
-The response is delivered to the callback `cb`, where the 1st argument is the
-possible error, and the 2nd argument is the created feed (which can be either
-the root meta feed `{ seed, keys }` or a sub feed
-`{ feedpurpose, subfeed, keys, metadata }`).
-
-### `sbot.metafeeds.findAndTombstone(metafeed, visit, reason, cb)`
+### `sbot.metafeeds.findAndTombstone(metafeed, find, reason, cb)`
 
 _Looks for the first subfeed of `metafeed` that satisfies the condition in
-`visit` and, if found, tombstones it with the string `reason`.
+`find` and, if found, tombstones it with the string `reason`.
 
 This is strictly concerned with meta feeds and sub feeds that **you own**, not
 with those that belong to other peers.
 
-`metafeed` must be a meta feed object `{ seed, keys }` (as returned by
-`getRoot()`).
-
-`visit` must be a function of the shape `({feedpurpose, subfeed, metadata, keys}) => boolean`.
-
-`reason` must be a string to describe why the found feed is being tombstoned.
+Arguments: 
+- `metafeed` *FeedDetail* object (as returned by e.g. `findOrCreate()`, `getRoot()`).
+- `find` *function* of the shape `(FeedDetail) => boolean`.
+- `reason` *String* - describes why the found feed is being tombstoned.
 
 The callback is called with `true` on the 2nd argument if tombstoning suceeded,
 or called with an error object on the 1st argument if it failed.
 
+
 ### `sbot.metafeeds.getRoot(cb)`
 
 Looks for the root meta feed declared by your main feed, and returns it (as
 `{ seed, keys}`) via the callback `cb` if it exists.
 
-If it does not exist, this API will **not** create the root meta feed.
+NOTE: If it does not exist, this API will **not** create the root meta feed.
```

Comment:
What's the purpose of "NOTE:"? Doesn't seem like it adds to the description.

Reply:
Happy to remove. I like Note of there's a perhaps expected or important side effect or behavior